packages/gatsby-plugin-google-tagmanager/.babelrc

```diff
@@ -0,0 +1,14 @@
+{
+  "presets": [
```

Comment:
Oh I just remembered that I put it there to fix a bug! Client side code needs it because otherwise it wont work in older browsers. The backticks will end up in the production bundle for example.